acceptance/bundle/deploy/dashboard/detect-change/script

```diff
@@ -0,0 +1,34 @@
+# MSYS2 automatically converts absolute paths like /Users/$username/$UNIQUE_NAME to
+# C:/Program Files/Git/Users/$username/UNIQUE_NAME before passing it to the CLI
+# Setting this environment variable prevents that conversion on windows.
+export MSYS_NO_PATHCONV=1
```

Comment:
nit: you can also use Env. MSYS_NO_PATHCONV="1" in test.toml.

Reply:
done in [0fb1c48](https://github.com/databricks/cli/pull/2918/commits/0fb1c487990c716731c7c01539df46266871a636)